src/applications/caregivers/hooks/useDefaultFormData.jsx

```diff
@@ -4,7 +4,7 @@ import { setData } from 'platform/forms-system/src/js/actions';
 import { selectFeatureToggles } from '../utils/selectors';
 
 export const useDefaultFormData = () => {
-  const { data: formData } = useSelector(state => state.form);
+  const formData = useSelector(state => state.form.data);
```

Comment:
Minor performance update here. We only want to rerender when `state.form.data` changes, not when `state.form` changes.